.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -514,8 +530,24 @@ public class RequiredActionUpdate : RunUpdate {
     public class RunCollectionOptions {
         public string AfterId { get; set; }
         public string BeforeId { get; set; }
-        public ListOrder? Order { get; set; }
-        public int? PageSize { get; set; }
+        public RunCollectionOrder? Order { get; set; }
+        public int? PageSizeLimit { get; set; }
```

Comment:
I am not sure we need the "limit" suffix. Don't we call it pageSize everywhere despote the fact that sometimes results are smaller? Also, if we really want to indicate that it's up to the size, them using "Max" is more .NET customary

Reply:
The main reason why I added the "Limit" suffix is because I would like to tie it back to the REST API, where this parameter is called "limit". We've had a couple of instances of users who knew that the service offered some functionality and that it was supposed to be called a certain way, but they weren't able to find it in our library because we renamed it and they thought it was missing. `PageSize` was one of those cases. By naming it to `PageSizeLimit`, my hope is to make the mapping more obvious.
 
I think this is a problem that we have in this library that we don't have in Azure libraries. Here, there is already a non-trivial amount of pre-existing knowledge of how the REST API works, mostly because all the existing documentation (including Python's and JavaScript's) is based on the REST API.